code/game/turfs/turf.dm

```diff
@@ -14,6 +14,10 @@
 	var/to_be_destroyed = 0 //Used for fire, if a melting temperature was reached, it will be destroyed
 	var/max_fire_temperature_sustained = 0 //The max temperature of the fire which it was subjected to
 
+	var/turf_heat_resistance = 100
+	var/turf_max_heat_resistance = 100
```

Comment:
Could you rename this to something like `heat_integrity`?

Reply:
sure can do